dts/bindings/flash_controller/emulated-flash-controller.yaml

```diff
@@ -0,0 +1,22 @@
+---
+title: flash controller Base Structure
+id: emulated-flash-controller
+version: 0.1
+
+description: >
+    This binding gives the base structures for all flash controller devices
```

Comment:
This binding gives the base structures for the emulated flash controller devices